Traceback (most recent call last):
  File "act_test.py", line 65, in <module>
    num_tran=act_test(session)
  File "act_test.py", line 45, in act_test
    a=transactions.new_order_transaction(c_id=new_order_c_id, w_id=new_order_w_id, d_id=new_order_d_id,M=new_order_m, items=new_order_line, current_session=session)
  File "/home/stuproj/cs4224f/playground/transactions.py", line 156, in new_order_transaction
    {'i_id': ol_i_id, 'w_id': ol_supply_w_id}
  File "cassandra/cluster.py", line 2030, in cassandra.cluster.Session.execute (cassandra/cluster.c:38536)
  File "cassandra/cluster.py", line 3844, in cassandra.cluster.ResponseFuture.result (cassandra/cluster.c:80834)
cassandra.ReadTimeout: Error from server: code=1200 [Coordinator node timed out waiting for replica nodes' responses] message="Operation timed out - received only 0 responses." info={'received_responses': 0, 'required_responses': 2, 'consistency': 'QUORUM'}
